CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.2.7](https://github.com/wessberg/di-compiler/compare/v2.2.6...v2.2.7) (2022-5-26)
+
+### Bug fixes
+
+* Fixed issue with generics to allow `container.register<IService<Generic>, Service<Generic>>()`.
+* Restricted `typescript` to minor version updates from `~4.6.4` as version 4.7 is currently incompatible.
```

Comment:
The changelog is actually auto-generated based on the commit log whenever a new version is released. So, unfortunately these two lines would actually be lost. To achieve the same result, you can write commit messages along these lines :)

Reply:
Sweet.  I'll back this out.

Reply:
Awesome!